docs/api-inference/index.md

```diff
@@ -1,61 +1,172 @@
-# Serverless Inference API
+# Inference Providers
 
-**Instant Access to thousands of ML Models for Fast Prototyping**
+The Hugging Face Inference Providers revolutionizes how developers access and run machine learning models by offering a unified, flexible interface to multiple serverless inference providers. This new approach extends our previous Serverless Inference API, providing more models, increased performances and better reliability thanks to our awesome partners.
 
-Explore the most popular models for text, image, speech, and more — all with a simple API request. Build, test, and experiment without worrying about infrastructure or setup.
+To learn more about the launch of the Inference Providers, check out our [announcement blog post](https://huggingface.co/blog/inference-providers).
 
----
+## Why use the Inference Providers?
 
-## Why use the Inference API?
-
-The Serverless Inference API offers a fast and simple way to explore thousands of models for a variety of tasks. Whether you're prototyping a new application or experimenting with ML capabilities, this API gives you instant access to high-performing models across multiple domains:
+The Inference Providers offers a fast and simple way to explore thousands of models for a variety of tasks. Whether you're experimenting with ML capabilities or building a new application, this API gives you instant access to high-performing models across multiple domains:
```

Comment:
Hammering my point from my previous comment - either:
- "(HF) Inference Providers offers" (as in "ProductX offers")
- "The Inference Providers {feature,API} offers"

but not "The Inference Providers offers" 😅 (same everywhere)

```suggestion
Inference Providers offers a fast and simple way to explore thousands of models for a variety of tasks. Whether you're experimenting with ML capabilities or building a new application, this API gives you instant access to high-performing models across multiple domains:
```

Reply:
100% aligned! 
Sorry about that, I didn't review it when I switched from "Inference Providers API" to "Inference Providers" :grimacing: